File: source/backend-vacatures-bron/src/main/java/com/cap/pocvng/entity/StraatadresBuitenland.java
package com.cap.pocvng.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.Embedded;
import javax.persistence.Entity;
import javax.validation.Valid;

/**
 * StraatadresBuitenland
 */


@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class StraatadresBuitenland extends AdresDetailsBuitenland {

    @Valid
    @Embedded
    private StraatadresBuitenlandImpl straatadresbuitenland;

}

File: source/frontend/src/app/entities/vacature/straatadresBuitenland.ts
import {StraatadresBuitenlandImpl} from './straatadresBuitenlandImpl';

export interface StraatadresBuitenland {
  straatadresbuitenland?: StraatadresBuitenlandImpl;
}


export function isStraatadresBuitenland(object: any): object is StraatadresBuitenland {
  return 'straatadresbuitenland' in object;
}
